language/stdlib/modules/Libra.move

```diff
@@ -215,22 +222,10 @@ module Libra {
         move_to(config_account, CurrencyRegistrationCapability{ cap })
     }
 
-    /// Publishes the `MintCapability` `cap` for the `CoinType` currency
-    /// under `account`. `CoinType`  must be a registered currency type.
-    /// The caller must pass a `TreasuryComplianceRole` capability.
-    public fun publish_mint_capability<CoinType>(
-        publish_account: &signer,
-        cap: MintCapability<CoinType>,
-        tc_account: &signer,
-    ) {
-        assert(Roles::has_treasury_compliance_role(tc_account), ENOT_TREASURY_COMPLIANCE);
-        assert_is_currency<CoinType>();
-        move_to(publish_account, cap)
-    }
-
     /// Publishes the `BurnCapability` `cap` for the `CoinType` currency under `account`. `CoinType`
     /// must be a registered currency type.
     /// The caller must pass a `TreasuryComplianceRole` capability.
+    /// TODO (dd): I think there is a multiple signer problem here.
```

Comment:
Yep. I think it would be ok to simplify this to a single signer for now (e.g., drop `account: &signer`) since it is always TC at the callsite (someone was actually asking about this here: https://community.libra.org/t/tc-account-in-coin1-move/2992). But that can be done in another PR, and there's also no harm leaving it how it is.